src/content/docs/query-your-data/explore-query-data/query-builder/query-builder-basic-mode.mdx

```diff
@@ -126,7 +126,7 @@ This example shows how to create a chart in basic mode.
     />
 
     <figcaption>
-      **one.newrelic.com > Query builder > Basic >** (event and attribute specified) **>** (filters and facets applied) **>** (time range and limit customized) **>** (chart type customized)
+      **[one.newrelic.com](https://one.newrelic.com) > Query builder > Basic >** (event and attribute specified) **>** (filters and facets applied) **>** (time range and limit customized) **>** (chart type customized)
```

Comment:
How about this: **[one.newrelic.com](https://one.newrelic.com) > Query builder > Basic > (event and attribute specified) > (filters and facets applied) > (time range and limit customized) > (chart type customized)